server/lib/packets/channel.go

```diff
@@ -3,15 +3,15 @@ package packets
 import (
 	"log"
 
-	proto "../../proto"
-	"../models"
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)